gym/core.py

```diff
@@ -39,6 +39,7 @@ class Env(object):
     # Set these in ALL subclasses
     action_space = None
     observation_space = None
+    is_vector_env = False
```

Comment:
I'm not a huge fan of overloading the `Env` API to handle `VectorEnv`. Ideally `VectorEnv` should be able to exist without having to touch too much the base API. I understand that this may be useful for wrappers to handle both regular `Env` and `VectorEnv`, but I would be more in favor to use `VectorEnvWrapper` in those cases that would require `VectorEnv`-specific features.

Reply:
That makes sense. I will find a workaround for this.